truss-chains/examples/rag/rag_chain.py

```diff
@@ -0,0 +1,144 @@
+# This file uses Truss Chains to create and deploy an LLM with access to a vector database
+
+import os
+
+import truss_chains as chains
+
+
+class VectorStore(chains.ChainletBase):
+
+    remote_config = chains.RemoteConfig(
+        docker_image=chains.DockerImage(pip_requirements=["chromadb"])
+    )
+
+    def __init__(self, context: chains.DeploymentContext = chains.depends_context()):
+        import chromadb
+
+        self._chroma_client = chromadb.EphemeralClient()
+        self._collection = self._chroma_client.create_collection(name="bios")
+
+        documents = [
```

Comment:
Inlining the content in the code, seems to distract from the functional structure. What about these alternatives:
* Move this to a module level constant at the top of the file (so it doesn't interrupt the rest of the code)?
* Move it into a text or csv file, and use the "data_dir" (like for trusses) to include and load it - that would abe also an instructive example of how to use "data_dir".